integration-libs/cdc/manage-users/cdc-user.config.ts

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
Please delete this file as it is unnecessary.


Minor changes in `cdc-user-details.module.ts` would suffice.

Reply:
done